src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -198,6 +199,9 @@
     <Reference Include="Microsoft.VisualStudio.CallHierarchy.Package.Definitions.dll">
       <HintPath>$(DevEnvDir)\Microsoft.VisualStudio.CallHierarchy.Package.Definitions.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.VisualStudio.Shell.Design.dll">
+      <HintPath>$(DevEnvDir)\Microsoft.VisualStudio.Shell.Design.dll</HintPath>
```

Comment:
i was copy/pasting the above line.  I can try referencing directly from teh SDK.